SPECS/influx-cli/influx-cli.spec

```diff
@@ -91,13 +77,13 @@ bin/influx completion zsh > %{buildroot}/%{_datadir}/zsh/site-functions/_influx
 %doc README.md CHANGELOG.md
 %{_bindir}/influx
 
-%files bash-completion
-%{_datadir}/bash-completion
```

Comment:
The file names may be the same but after a quick check it looks like the contents differ. Have you checked, that we're not losing some functionality with this change?

Reply:
This looks like [an interesting tutorial on how bash-completion works](https://opensource.com/article/18/3/creating-bash-completion-script) - it helped me to understand the basics.

Reply:
That is a very good point, thank you for finding this. I think for now we want to remove the package to unblock installation of `influx-cli`. But I will write this into the bug assigned to this and keep the bug open and keep investigating about it.